spec/stat_tracker_spec.rb

```diff
@@ -24,14 +24,53 @@
     it 'exists' do
       expect(@stat_tracker).to be_instance_of(StatTracker)
     end
-end
-# Game Statistics Tests
 
+    describe '#count of teams' do
```

Comment:
Is this indentation a little off? I can't tell for sure on GH

Reply:
Took a look at it in my IDE and I think my 'exists' test block was in one indent too far, but I also didn't put a separate describe block for it and I think that would make it look more uniform and easier to read with the rest of the code. I'll go head and throw that in there and let me know if it looks better!

Reply:
Thank you. Yeah, it's probably on my end, not yours.